training/train.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+
+import torch
+import torchvision
+import torchvision.transforms.v2 as transforms
+from torch.utils import data
+
+from data.dataset import Dataset
+from models.centernet import ModelBuilder, input_height, input_width
+from training.encoder import CenternetEncoder
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-o", "--overfit", action="store_true", help="overfit to 10 images")
+args = parser.parse_args()
+
+overfit = args.overfit
+
+dataset_val = torchvision.datasets.VOCDetection(
+    root="../VOC", year="2007", image_set="train", download=False
+)
+
+transform = transforms.Compose(
+    [
+        transforms.Resize(size=(input_width, input_height)),
+        transforms.ToImage(),
+        transforms.ToDtype(torch.float32, scale=True),
+    ]
+)
+
+encoder = CenternetEncoder(input_height, input_width)
+
+dataset_val = torchvision.datasets.wrap_dataset_for_transforms_v2(dataset_val)
+torch_dataset = Dataset(dataset=dataset_val, transformation=transform, encoder=encoder)
+
+training_data = torch_dataset
+lr = 0.03
+desired_loss = 0.1
+batch_size = 32
+
+if overfit:
+    # 10 randomly chosen pictures for model training
+    trainingdata_indices = torch.randperm(len(torch_dataset))[:10]
```

Comment:
it is better to fix the first 10 images because we need to visualize the result. However, the result could be verified only on the same images on which the model is overfitted.

Reply:
Changed training dataset to the first 10 images for the overfit case.